libminifi/test/tensorflow-tests/TensorFlowTests.cpp

```diff
@@ -19,14 +19,14 @@
 #include <memory>
 #include <string>
 
-#include <tensorflow/cc/framework/scope.h>
-#include <tensorflow/cc/ops/standard_ops.h>
-#include <processors/PutFile.h>
-#include <processors/GetFile.h>
-#include <processors/LogAttribute.h>
+#include <processors/GetFile.h> // NOLINT
+#include <processors/LogAttribute.h> // NOLINT
+#include <processors/PutFile.h> // NOLINT
+#include <tensorflow/cc/framework/scope.h> // NOLINT
+#include <tensorflow/cc/ops/standard_ops.h> // NOLINT
```

Comment:
Regarding my previous suggestion: I suggest trying to convert our headers to use quotation marks and leave 3rd parties as is, even if it means that we need to use NOLINT.

Reply:
Will do that in a separate PR, right? I don't want to rebase my linter PR again.

Reply:
My initial idea was to do it here, but you're right, it's better not to mix in this.